pages/wg/website.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Website Working Group
+subtitle:
+permalink: /wg/website/
+---
+
+# Overview
+
+The website working group was established to work on the website that you are reading right now! To get involved with the website working group, visit the [`#website`](https://usrse.slack.com/messages/website) channel on the US-RSE slack. This page include some additional information about content on the site.
+
+
+## Feeds
+
+Added in early 2022, we wanted an ability to provide easy ways to automate specific events, or subscribe
+to them. For the reason, we added differet kinds of programmatic subscriptions or feeds that include:
+
+ - rss feeds for different kinds of content
+ - equivalent json feeds for the same.
+ 
+The rss feeds are intended to be subscribed to by an appropriate service, while the json API is
+more suitable for a client tool.
+
+### RSS (xml) Feeds:
+
+For each RSS feed, we provide the latest 100 items.
+
+ - [https://us-rse.org/feeds/events.xml](https://us-rse.org/feeds/events.xml): subscribe to all US-RSE events.
+ - [https://us-rse.org/feed.xml](https://us-rse.org/feed.xml): subscribe to posts on the US-RSE site
```

Comment:
Same with this list. I'd suggest putting feed.xml as the first item.

Reply:
Also done locally!